### DISCUSSION TITLE
OverlayScrollbars scroll speed

### DISCUSSION THREAD

**Discussion by MaxAlekseyev** [KingSora/OverlayScrollbars#326] [Q&A]
Hello,

Can I adjust scroll speed using some native OverlayScrollbars methods?

Regards, Max

**Comment by KingSora** [comment]
Good day @MaxAlekseyev 

Nope, thats not possible since OverlayScrollbars is using the systems scroll behavior 1:1, because its just hiding the native scrollbars.
If you wanna change the scroll behavior (speed, smoothness etc.) you still can use other Javascript plugins for that. They should work in combination with OverlayScrollbars, their initialization could be a bit different though.

**Reply by MaxAlekseyev** [reply]
Thank you for quick answer. I'll use third parties plugin for this. The OverlayScrollbars is perfect anyway, great work.
